messages/window/window.ar.yml

```diff
@@ -0,0 +1,13 @@
+kendo:
+  window:
+    # The title for the close button
```

Comment:
This should be "The title **of** the close button" so it's in line with the rest of the message descriptions.